### PR TITLE
Make paths in metadata viewer clickable

### LINKED ISSUE #582: [MetadataViewer] Navigation for NeXus attributes
Make the values of any `default`, `signal`, `axes`, etc. attributes clickable in the `MetadataViewer` so users can quickly navigate to the entities.

### PR DESCRIPTION
**Pull request by axelboc**
Part of #582 

https://user-images.githubusercontent.com/2936402/114418536-e564bb00-9bb2-11eb-8fa2-dc1f1fc606cc.mp4

I haven't made attributes with array values clickable yet.

### CODE REVIEW

File: src/h5web/metadata-viewer/AttributesTable.tsx [hunk 1]
```diff
@@ -25,7 +34,21 @@ function AttributesTable(props: Props) {
         {attributes.map(({ name, value }) => (
           <tr key={name}>
             <th scope="row">{name}</th>
-            <td>{JSON.stringify(value)}</td>
+            <td>
+              {FOLLOWABLE_ATTRS.has(name) && typeof value === 'string' ? (
```
Comment (axelboc) [inline src/h5web/metadata-viewer/AttributesTable.tsx:38]: I only deal with `string` attributes for now. Will deal with array of strings next.

Comment (loichuder) [reply src/h5web/metadata-viewer/AttributesTable.tsx:38]: We might want to extract this in its own component then.

**Review by loichuder — APPROVED** [review]
Nice !

A simple question: would it be possible to dynamically check if an attribute is resolvable by checking in the `entityStore` ?

**Comment by axelboc** [comment]
> A simple question: would it be possible to dynamically check if an attribute is resolvable by checking in the `entityStore` ?

Well, the `entityStore` doesn't know anything about the entities that haven't been visited yet, so it wouldn't be of much use. 

The `FOLLOWABLE_ATTRS` constant introduces a bit of an implicit coupling with NeXus, but at least it's simple and easy to understand, so I think it's reasonable for now.

**Comment by loichuder** [comment]
> Well, the entityStore doesn't know anything about the entities that haven't been visited yet, so it wouldn't be of much use.

True, I forgot it only stores visited entities. I just wish we could notify the user if/when the path is not reachable.

**Comment by axelboc** [comment]
I've fixed the regression and added a feature test.

**Pull request merged**